app/views/users/edit.html.erb

```diff
@@ -48,7 +48,9 @@
           </span>
         </h1>
       <% else %>
-        <h1 class="fs-xl s:fs-3xl">Settings for <a href="/<%= @user.username %>">@<%= @user.username %></a></h1>
+        <h1 class="fs-xl s:fs-3xl">
+          Settings for <a href="<%= user_url(@user) %>">@<%= @user.username.truncate(User::USERNAME_MAX_LENGTH) %></a>
```

Comment:
This is to truncate the displaying of the username in case the user manages to send more than 30 chars (which breaks the layout)